### ISSUE TITLE
no error for `collections.abc.MutableMapping` is not subscriptable

### ISSUE THREAD

**Issue by KotlinIsland** [python/mypy#11529]
```py
from collections.abc import MutableMapping

MutableMapping[int, int]
```
`mypy python-version 3.6 test.py`: `Success: no issues found in 1 source file`
`python test.py`:
```
TypeError: 'ABCMeta' object is not subscriptable
```

**Comment by sobolevn** [comment]
Related https://github.com/python/typeshed/issues/6257
